common/data_refinery_common/migrations/0019_auto_20190404_1427.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.1.5 on 2019-04-04 14:27
+
+from django.db import migrations
+from data_refinery_common.utils import get_env_variable
+
+
+# We want this to throw if it can't access this, no point in running a
+# migration to set everything to a bad value.
+S3_BUCKET_NAME = get_env_variable("S3_QN_TARGET_BUCKET_NAME")
```

Comment:
Would prefer this to match `S3_QN_TARGET_BUCKET_NAME`

Reply:
This is a good point!